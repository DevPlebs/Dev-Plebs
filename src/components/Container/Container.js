import styled from "@emotion/styled"
import tw from "tailwind.macro"

export const Container = styled.div`
  ${tw`max-w-5xl mx-auto`}
`

export default Container
