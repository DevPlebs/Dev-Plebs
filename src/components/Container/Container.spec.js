import React from "react"
import { create } from "react-test-renderer"

import Container from "./Container"

describe("Container", () => {
  it("renders correctly", () => {
    const tree = create(<Container />)
    expect(tree.toJSON()).toMatchSnapshot()
  })
})
