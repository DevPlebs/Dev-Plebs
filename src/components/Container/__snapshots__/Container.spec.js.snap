// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders correctly 1`] = `
.emotion-0 {
  max-width: 64rem;
  margin-left: auto;
  margin-right: auto;
}

<div
  className="emotion-0"
/>
`;
